charts/osdfir-infrastructure/README.md

```diff
@@ -240,6 +253,65 @@ Follow these steps to upgrade the database on your Kubernetes deployment:
       kubectl rollout restart deployment my-release-timesketch-web
       ```
 
+### Managing the GRR and Fleetspeak config
+
+The Fleetspeak frontend for GRR can be exposed in three different modes:
```

Comment:
Could we change this to something along the lines of `The main way that the Fleetspeak frontend for GRR is exposed is through a NodePort. This allows the connection to be port-forwarding into the local machine running.

For learning about other ways to expose GRR within GCP, see the install-on-gke.md guide...

And then we could move the rest of the section into that guide. The reason being is to keep at least this README as vanilla as possible but then have a dedicated guide/install for the GCP implementation.

Reply:
Maybe for now what we could do is add number 2 and 3 into the bottom of `installing-on-gke.md` then in a second iteration once we update that doc for GRR have it fit in better.

Reply:
Good points.
Updated as suggested.